src/com/epam/javacourse/homework4/HousePlant.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.javacourse.homework4;
+
+public class HousePlant implements Plant {
+    String name;
+    String type;
+    String nativeRegion;
+
+    @Override
+    public void createPlant(String name, String type, String nativeRegion) {
```

Comment:
It's common to return an instance of a created object in 'create' methods.
When you create a new plant you return a `new Plant`. The thing you do here can be naturally done with `getters` and `setters` from the previous homework.